.github/workflows/publish_patch.yaml

```diff
@@ -0,0 +1,29 @@
+name: Publish Patch
+
+# Run 15 minutes after the hour, every hour.
+# We use this instead of the top of the hour to avoid heavy-traffic times on github.
```

Comment:
Lol.  They really should have a setting for that.  We don't really care when it runs so much as that it runs about every X time alotments.